app/src/components/common/footer/FooterInfo.js

```diff
@@ -12,16 +12,19 @@ export default function FooterInfo() {
             chaletinteriordesign@gmail.com
           </p>
         </div>
-        <div className="">
+        <div>
```

Comment:
if divs are empty, do you really need them?

Reply:
this div is needed to apply the flexbox effect